src/server/api/controllers/qr.controller.js

```diff
@@ -0,0 +1,22 @@
+const knex = require('../../config/db');
+
+const get = async (id) => {
+  try {
+    const instances = await knex('game_instances')
+      .select('game_instances.game_code as code')
+      .where({ id });
+
+    if (instances.length === 0) {
+      throw new Error('There is no record');
+    }
+    return {
+      qr: instances[0].code,
```

Comment:
you can make it more clear like this
QRcode: instances[0].QRcode,

Reply:
Hey Amer, code is one of the column here based on diagram for game_instance table , so I can't use any thing different from that name:) you can have a look to diagram here in side the isuue [https://github.com/HackYourFuture-CPH/fp-class12/issues/83](url) and also here  [https://dbdiagram.io/d/5ee687a19ea313663b3a885c](url) at code documentation

Reply:
yeah that's true, but if you look at the previous comment in line 6 then it should work perfectly , i already tested it locally , because you already assign the 'game_instances.game_code as code' 🙂

Reply:
<img width="690" alt="Screenshot 2020-06-24 at 14 12 14" src="https://user-images.githubusercontent.com/54769123/85554383-20e2c880-b625-11ea-9bb9-c3c387cbc180.png">